### ISSUE TITLE
⌨  Accessibility Issue: Profile and Application navigation is unnecessarily complex

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#7530]
## ⌨ Accessibility Issue

Navigating from the Profile and applications (user dashboard) page to add an experience is too complex. First, users do not understand that they must activate the "Career timeline and recruitments" link to add or edit an experience. Second, when a user lands on this page, it needs to be clarified that the goal is for them to add or edit their career experiences.   

## 🦋 Expected Behaviour

1. When a user lands on the dashboard page (Profile and applications), it is obvious that they can manage their system profile information, career timeline, skills showcase, and view the recruitment processes to which they have applied.
2.  

What should be happening?

## 🕵️ Details

Add any additional details that could assist with troubleshooting/fixing the issue.

- **Operating System**: Name and version (if possible).
- **Browser**: Name and version (if possible).
- **Assistive Technology**: Name and version (if possible).

## 📋 Steps to Reproduce

1. ...
2. ...

## 📸 Screenshot

Add a screenshot (if possible).

## 🙋‍♀️ Proposed Solution

(optional) Do you have a proposed solution?

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Criteria 1
- [ ] Criteria 2

**Comment by substrae** [comment]
Couple things to note here in terms of the WCAG and acceptance criteria:
- 2.4.5 refers to meeting two or more of the proposed solutions, of which we already do; if we want to expand that further, we could look at producing and linking to a sitemap, something we should already be doing anyway
  - providing links to pages (the card title and the add/edit button)
  - providing a table of contents (the card's content)
- users can already locate adding/editing an experience multiple ways - through their profile, or through an application; again though, the sitemap would add a third path
- in terms of it being clear how to add things - we could link the individual card elements to the experience section on the career timeline, but it might be more confusing then helpful for people if the list isn't filtered by the type they clicked on
- the user doesn't currently have to click 3 times to add or edit - they click once on their career timeline card, and again on the add or edit buttons
- I'm not sure how to make the buttons more clear to SR users once they're already on the page, but open to suggestions here - right now they navigate to the "Manage your career timeline" section and it's the first interactive element they hit
  - There's no easier way to reach the edit button, because it exists within an experience and requires that context

This all plays back into the conversation around how the "Career timeline" card is labelled - we know that "Your resume and experience" is clearer for users, and that would make it obvious where experiences are managed if the user hasn't read the table of contents directly below yet.

In terms of design direction, right now I think we only really need to do the following:
- [ ] Create a sitemap and link to it from the footer
- [ ] Continue the conversation around "Career timeline" and effective alternatives

**Comment by RM-1978** [comment]
@substrae, this success criterion intends to make it possible for users to locate content in a manner that best meets their needs. 

Users may find one technique more accessible or more comprehensible to use than another. 

At present, we are saying that three clicks are needed:
> - From profile and applications, click "Career timeline"
> - Click "Add an experience" to invoke dialog
> - Find the type of experience and click button

Because the ideal scenario might be to allow users to adjust their career timeline in 1-2 clicks from the profile and applications page ( ideally with only one context change), are there any risks to adding an "Add an experience" link to the Career timeline card?
